guess_a_no/README.md

```diff
@@ -0,0 +1,37 @@
+Game: Guess A Number
+=======
+
+### Objective
+
+Objective is to build a fun game, which will help a User to get familar with random no and basic logic. It's a small script and any language can be used for coding.
+

```

Comment:
What about building the same project with this theme?

Sherlock has been locked inside a room by Moriarity. Sherlock needs to open the door but to open a door sherlock needs to enter the correct key in the lock. 
You have to generate the program to help sherlock in opening the lock.

Some nice graphics if possible can also be placed in the quiz.